Common/UnitDefinitions/MassConcentration.json

```diff
@@ -238,6 +238,30 @@
           "Abbreviations": [ "ppg (U.S.)" ]
         }
       ]
+    },
+	{
+      "SingularName": "OuncePerUSGallon",
+      "PluralName": "OuncesPerUSGallon",
+      "FromUnitToBaseFunc": " x/0.1335264711843",
+      "FromBaseToUnitFunc": "x*0.1335264711843",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "opg (U.S)" ]
```

Comment:
I'm not familiar with these, but from google it seems this unit abbreviation is more widely used: `oz/gal`

If so, maybe instead of `opg` we could do `oz/gal (U.S.)` and `oz/gal (imp.)`. 

This is also consistent with `gal (U.S.)` and fluid ounce `oz (U.S.)` in `Volume`:
https://github.com/angularsen/UnitsNet/blob/c168b8975b7fd5391babf58829762514e37db570/Common/UnitDefinitions/Volume.json#L252
https://github.com/angularsen/UnitsNet/blob/c168b8975b7fd5391babf58829762514e37db570/Common/UnitDefinitions/Volume.json#L268


https://www.convert-me.com/en/convert/density/ozpgal.html?u=ozpgal&v=1
https://www.translatorscafe.com/unit-converter/en-US/density/17-33/gram/liter-ounce/gallon%20(US)/